docs/README.md

```diff
@@ -0,0 +1,108 @@
+@interlay/polkabtc / [Exports](/modules.md)
+
+# PolkaBTC JS
+
+JavaScript library to interact with PolkaBTC
+
+## Getting started
+
+See [BTC Parachain](https://github.com/interlay/btc-parachain) to get a development node running.
```

Comment:
I think the first instruction here should be the docker-compose since that's easier than to setup all the components individually.